db/db.go

```diff
@@ -3015,40 +3018,28 @@ func GetBLSChanges(query string, length, start uint64, orderBy, orderDir string)
 	var err error = nil
 
 	if trimmedQuery != "" {
-		joinQuery := `
-			LEFT JOIN (
-				SELECT 
-					validators.validatorindex as validatorindex,
-					eth1_deposits.from_address_text as deposit_address_text
-				FROM validators 
-				INNER JOIN eth1_deposits ON validators.pubkey = eth1_deposits.publickey
-			) AS val ON val.validatorindex = bls.validatorindex`
-
 		// Check whether the query can be used for a validator, slot or epoch search
 		if uiQuery, parseErr := strconv.ParseUint(query, 10, 64); parseErr == nil {
 			searchQuery := `
 				WHERE bls.validatorindex = $3			
 					OR block_slot = $3
-					OR (block_slot / $5) = $3
-					OR pubkey_text LIKE ($4 || '%')
-					OR deposit_address_text LIKE ($4 || '%')`
```

Comment:
**question:** Why was the address search removed here? Are there not valid addresses that can be parsed as **uint64**?